var/spack/repos/builtin/packages/hiredis/package.py

```diff
@@ -29,17 +24,53 @@ class Hiredis(CMakePackage, MakefilePackage):
     version("0.13.3", sha256="717e6fc8dc2819bef522deaca516de9e51b9dfa68fe393b7db5c3b6079196f78")
     version("0.13.2", sha256="b0cf73ebe039fe25ecaaa881acdda8bdc393ed997e049b04fc20865835953694")
 
+    build_system(
```

Comment:
I think `1:` and `:0` work better, right?  That way if 0.14.2 comes out it can be covered.

Reply:
Yes, good point! I have switched to `1:` and `:0`.